sample/F8/speakers

```diff
@@ -0,0 +1,1613 @@
+[
+   {
+      "id":2,
+      "name":"Michael Slater",
+      "email":"",
+      "mobile":"",
+      "photo":"",
+      "organisation":"Facebook",
+      "sessions":[
+         {
+            "id":5,
+            "title":"Introduction to the Camera Effects Platform"
+         }
+      ],
+      "country":"",
+      "short_biography":"",
+      "website":"",
+      "twitter":"",
+      "facebook":"",
+      "github":"",
+      "linkedin":"",
+      "position":"Product Manager",
+      "city":"",
+      "gender":"male",
+      "heard_from":"",
+      "icon":"",
+      "small":"",
+      "speaking_experience":"",
+      "sponsorship_required":"",
+      "thumbnail":""
```

Comment:
@arp95 there is no information on the official fbf8 site regarding this. i don't think it makes sense to add various types of information of the speakers if the site hasn't disclosed it.